### ISSUE TITLE
Support Accessibility regarding Coloring and Design

### ISSUE THREAD

**Issue by ansarmie** [SeleniumHQ/selenium-ide#965]
## 🐛 Bug Report

Fulfill the **W3 Standards** **1.4.3 - Contrast (Minimum)** and **1.4.11 - Non-text Contrast** (Recommendation) to support the use of the application by people with Hard of sight and Keyboard or Screen Reader users.

More info here:

- [Guideline 1.4 Distinguishable](https://www.w3.org/TR/WCAG21/#distinguishable)
- [Understanding Success Criterion 1.4.3: Contrast (Minimum)](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)
- [Understanding Success Criterion 1.4.11: Non-text Contrast](https://www.w3.org/WAI/wcag21/understanding/non-text-contrast.html)

## To Reproduce

- Color contrast ratio in Name your project and Controlled by Selenium IDE is less than 4.5:1.

## Expected behavior

- The text in a control, label or title should be at least of 4.5:1.

## Project file reproducing this issue (highly encouraged)

This is a issue with the UI, not of a specific Selenium scenario.

## Environment

OS: Windows 10
Selenium IDE Version:  3.16.1
-->
Browser: Google Chrome
Browser Version: 80.0.3987.116 (Official Build) (64-bit)

**Comment by tourdedave** [comment]
Fixed in [v3.17.0](https://github.com/SeleniumHQ/selenium-ide/releases/tag/v3.17.0)
